frontend/src/components/editor-page/app-bar/app-bar.tsx

```diff
@@ -47,6 +48,7 @@ export const AppBar: React.FC<AppBarProps> = ({ mode }) => {
             <ReadOnlyModeButton />
           </ShowIf>
           <HelpButton />
+          <CheatsheetButton />
```

Comment:
For now this is fine, but with the re-design this will be placed inside the "?"-dropdown, right?

Reply:
Yes